### DISCUSSION TITLE
Welcome to Moviesy Discussions!

### DISCUSSION THREAD

**Discussion by KaustubhPatange** [KaustubhPatange/Moviesy#22] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

If there are features you want from the app you can discuss them here with the maintainer. Once the feature request is accepted the discussion will be converted to the GitHub issue & development will be discussed over there.

Also, if you want to have a quick chat you are most welcome.

**Comment by cppLawyer** [comment]
Hey KaustubhPatange,

I was thinking, why don't you make this application compatible with Android TV then this app will be really fantastic. I also wanted to make one but with QML and C++ so the NDK but I see that your release with Kotlin works fine. Let me know if you're going to do it, it does not seem like a big change in the src code, also I would keep the app free, we're a community here on GitHub everyone watches and uses each other's code that makes us different from others . But in the end it is your own choice.

Kind Regards,

cppLawyer

**Comment by KaustubhPatange** [comment]
Yeah, when I initially made the application I didn't have Android TV to test the application or a good PC to run a TV emulator so I locked the app to only mobile devices but I'm pretty sure some people are still using this app on their Android TV because I was able to see the devices on which the app was installed.

Anyway, I added the casting feature in one of the future updates where you can cast the movie to the TV so that was helpful. Also, I don't think I'm going to add complete support for Android TV or Wear OS. If you want to do it, you can fork the repo, make changes & submit a PR.

You can also maintain your own version of Moviesy, if you do so just put the appropriate credits to the original app in your app about section as well as in the Github Readme section.

Cheers,
Kaustubh Patange

**Reply by cppLawyer** [reply]
Hello,

How about i make a cross platform version of Moviesy for Windows, Linux, MacOS, and you make a Android TV Release with full features. Let me know what you thing about this. And ofcourse i will give you the appropriate credits, btw i was the person who tested it on Android TV :-).

Kind Regards,

cppLawyer

**Reply by KaustubhPatange** [reply]
Well, I can't say dates because I'm busy with my job & other projects too. But whenever I get any motivation I'll surely work on that :)

**Comment by ghost** [comment]
Hello KaustubhPatange,

I recently downloaded this and was able to watch any movie perfectly without any issue, but I found out that it doesn't work with a WIFI connection - I was wondering why, since the downloading requires a lot of data, using a WIFI is better for downloading faster and is easier (But mainly since I don't use mobile data often). I'd just like to know whether this is a fault in phone where a setting is not allowing for the use of WIFI or just in general.

Kind Regards,
BoiledEggPie
